### PR TITLE
Add inventory_transfer command

### PR DESCRIPTION
**Pull request by Infernum1**

### CODE REVIEW

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]
1. This doesn't remove items from the `from_user`
2. It would be nice to have a "summary" of items moved just in case, I think a separate command which is called `!!debuginv <user>` would be nice and it lists out their items like:
```
({user_id}, {item_name}, {sell_price}, {amount}, {sticky}, {sellable}),
...
```
for easy db inserts

File: bot/cogs/commands/owner.py [hunk 1]
```diff
@@ -290,6 +290,19 @@ def format_row(idx_row) -> str:
...
+    async def transfer_inventory(self, ctx: Ctx, from_user: discord.User, to_user: discord.User):
+        item_count = 0
+        for item in await self.db.fetch_items(from_user.id):
+            if item.sticky or not item.sellable:
+                continue
+
+            item_count += 1
+            await self.db.add_item(to_user.id, item.name, item.sell_price, item.amount, item.sticky, item.sellable)
+            
+        await ctx.send(f"{self.d.emojis.yes} Transferred **{item_count}** items from {from_user.mention} to {to_user.mention}.")
```
Comment (Iapetus-11) [inline bot/cogs/commands/owner.py]: Can this be a reply instead

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]
1. This doesn't remove items from the `from_user`
2. It would be nice to have a "summary" of items moved just in case, I think a separate command which is called `!!debuginv <user>` would be nice and it lists out their items like:
```
({user_id}, {item_name}, {sell_price}, {amount}, {sticky}, {sellable}),
...
```
for easy db inserts

**Comment by Infernum1** [comment]
> 1. This doesn't remove items from the `from_user`
> 2. It would be nice to have a "summary" of items moved just in case, I think a separate command which is called `!!debuginv <user>` would be nice and it lists out their items like:
> ```
> ({user_id}, {item_name}, {sell_price}, {amount}, {sticky}, {sellable}),
> ...
> ```
> for easy db inserts

Oh sure I totally forgot about that, also the two things you mentioned (a summary and another debug command) are seperate right? Just a lil confused about that

File: bot/cogs/commands/owner.py (hunk 1 above)
Comment (Infernum1) [reply bot/cogs/commands/owner.py]: Sure sure

File: bot/cogs/commands/owner.py [hunk 2]
```diff
@@ -290,6 +290,26 @@ def format_row(idx_row) -> str:
             f"**Command Streaks** (last week)\n```md\n## d  h  m  s  | user id             | name\n{formatted_rows}\n```"
         )
 
+    @commands.command(
+        name="transfer_inventory", aliases=["invtransfer", "auction", "transferinv", "trinv"]
+    )
+    @commands.is_owner()
+    async def transfer_inventory(self, ctx: Ctx, from_user: discord.User, to_user: discord.User):
```
Comment (Iapetus-11) [inline bot/cogs/commands/owner.py]: Can you make this take `int | discord.User` so that this can handle users which don't share any servers with the bot?

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

**Review by Iapetus-11 — APPROVED** [review]

**Comment by Iapetus-11** [comment]
Ty

**Pull request merged**